src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -364,9 +364,9 @@ def prepare_logs_argument(c):
             c.argument(
                 'registry_password', help='The password of the container registry.', arg_group='Custom Container')
             c.argument(
-                'container_command', help='The command of the container image.', nargs='*', arg_group='Custom Container')
+                'container_command', help='The command of the container image.', arg_group='Custom Container')
             c.argument(
-                'container_args', help='The arguments of the container image.', nargs='*', arg_group='Custom Container')
+                'container_args', help='The arguments of the container image.', arg_group='Custom Container')
```

Comment:
I would like to ask whether the change will cause breaking change to users?

Reply:
No. Actually the correct type is `string`. It's changed to the array accidentally in a previous refactoring.

Reply:
OK